src/components/ProtectedImage.vue

```diff
@@ -0,0 +1,26 @@
+<script lang="ts" setup>
+import filters from "@/utils/filters";
+import { useAuth0 } from "@auth0/auth0-vue";
+import { onMounted, ref } from "vue";
+
+const props = defineProps<{
+  src: string | null | undefined;
+}>();
+
+const source = ref("");
+
+// TODO: fix this for tests (auth0 isn't in context in tests at this time)
```

Comment:
Should be fairly easy to do using a `spy` as I've done in `PlaylistOverview.spec.ts`